### ISSUE TITLE
PHP 8.2 Deprecation warnings

### ISSUE THREAD

**Issue by onigoetz** [dauxio/daux.io#357]
```
Deprecated: Creation of dynamic property Todaymade\Daux\Config::$templateRenderer is deprecated in /daux/libs/Format/HTML/Generator.php on line 34
Deprecated: Function utf8_encode() is deprecated in /daux/libs/DauxHelper.php on line 105
Deprecated: Creation of dynamic property Highlight\RegExMatch::$type is deprecated in /daux/vendor/scrivo/highlight.php/Highlight/Highlighter.php on line 116
Deprecated: Creation of dynamic property Highlight\RegExMatch::$rule is deprecated in /daux/vendor/scrivo/highlight.php/Highlight/Highlighter.php on line 117
Deprecated: Creation of dynamic property Todaymade\Daux\Extension\Markdown\DocumentRenderer::$daux is deprecated in /tmp/daux/daux/Markdown/DocumentRenderer.php on line 18

```

**Comment by onigoetz** [comment]
Highlight will fix this soon : https://github.com/scrivo/highlight.php/issues/99

**Comment by onigoetz** [comment]
Fixed all deprecations under our control in 0.21.0 https://github.com/dauxio/daux.io/releases/tag/0.21.0
